### DISCUSSION TITLE
Recommended way to update / uninstall platform specific dependencies?

### DISCUSSION THREAD

**Discussion by runrc** [cogentcore/core#992] [Q&A]
We can run `core setup` to install platform specific dependencies (Vulcan SDK, etc).

What is the recommended way to,
1) update these platform specific dependencies?
2) uninstall these platform specific dependencies?

**Comment by kkoreilly** [comment]
That is definitely a good idea; I will add `core setup -update` and `core setup -uninstall` commands to do those things soon.

**Reply by kkoreilly** [reply]
Also please note that `core setup` commands should never be run on end user devices; `core pack` creates an app package and/or installer that handles everything automatically (I still need to implement `.msi` generation on Windows, but otherwise it is all working).

**Comment by kkoreilly** [comment]
As part of this we should also consider how to best support easily/automatically updating the core tool.
